Logger/Book.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Logger;
+
+public record Book : IEntity
+{
+    // implemented implicitly since Name is a frequently-accessed member
+    public string Name { get; set; }
+
+    //TODO: explain why this is implemeted implicitly
+    public Guid Id { get; init; }
+
+    private FullName _author;
+        public string Author
+    {
+        get => _author.ToString();
+        set => _author = new(value);
+    }
```

Comment:
```suggestion
      public FullName Author {get; set;}
```
A auto-generated property seems simpler here. Also, what is the purpose of this, is it used anywhere?